HOWTO.md

```diff
@@ -8,8 +8,7 @@
 
 ### Desktop
 
-1. Start the dev server `make dev-server`
-2. Run `make dev-client`
+1. Run from source with `pnpm start`
```

Comment:
This makes the following note confusing.

Reply:
Yes, fixed